fusor-ember-cli/app/controllers/deployment-new/satellite/configure-environment.js

```diff
@@ -45,7 +45,7 @@ export default Ember.Controller.extend(ConfigureEnvironmentMixin, {
       var environment = this.store.createRecord('lifecycle-environment', this.get('fields_env'));
       environment.save().then(function(result) {
         //success
-        self.get('nonLibraryEnvironments').pushObject(result);
+        self.get('lifecycleEnvironments').addObject(result);
```

Comment:
@jeff-phillips-18, FYI, I'm not sure why `pushObject` didn't work. I thought they were the same.